### PR TITLE
Version Bump in master

### PR DESCRIPTION
**Pull request by AndyGerlicher**
Moving from 14.1 to 15.0.

Also adding pkgdef and a setup proj for our internal builds. Still a work in progress at this point.

### CODE REVIEW

**Comment by AndyGerlicher** [comment]
@dotnet-bot test this please

**Comment by AndyGerlicher** [comment]
@dotnet-bot test this please

File: CreateSetupPackageLayout.proj [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  <Import Project="dir.props" />
+
+  <PropertyGroup>
+      <SetupPackageLayoutDir>$(BinDir)Setup\</SetupPackageLayoutDir>
+      <SetupPackageOutputFile>$(SetupPackageLayoutDir)MSBuild-master-0001.fopx</SetupPackageOutputFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Commenting out files that aren't produced by us
+    (When comparing against Program Files\MSBuild\14.0\Bin) -->
```
Comment (rainersigwald) [inline CreateSetupPackageLayout.proj]: >. . . from a VS2015 Update 2 install.

(or whatever)

File: CreateSetupPackageLayout.proj [hunk 2]
```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="dir.props" />
+
+  <PropertyGroup>
+      <SetupPackageLayoutDir>$(BinDir)Setup\</SetupPackageLayoutDir>
+      <SetupPackageOutputFile>$(SetupPackageLayoutDir)MSBuild-master-0001.fopx</SetupPackageOutputFile>
```
Comment (rainersigwald) [inline CreateSetupPackageLayout.proj]: Does this 0001 number need to be parameterized?

File: CreateSetupPackageLayout.proj [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Commenting out files that aren't produced by us
```
Comment (rainersigwald) [inline CreateSetupPackageLayout.proj]: "us" == open source build?

File: CreateSetupPackageLayout.proj [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.ServiceModel.targets" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.VisualBasic.Core.targets" /> -->
+    <BinX86Output Include="$(OutputPath)Microsoft.VisualBasic.CurrentVersion.targets" />
+    <BinX86Output Include="$(OutputPath)Microsoft.VisualBasic.targets" />
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.WinFx.targets" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.WorkflowBuildExtensions.targets" /> -->
+    <!-- <BinX86Output Include="$(OutputPath)Microsoft.Xaml.targets" /> -->
+    <BinX86Output Include="$(OutputPath)MSBuild.exe" />
+    <BinX86Output Include="$(OutputPath)MSBuild.exe.config" />
+    <!-- <BinX86Output Include="$(OutputPath)MSBuild.rsp" /> -->
```
Comment (rainersigwald) [inline CreateSetupPackageLayout.proj]: We should make + ship this `.rsp` file, right?

File: src/Utilities/Microsoft.Build.Utilities.Core.pkgdef [hunk 5]
```diff
@@ -0,0 +1,6 @@
+[$RootKey$\RuntimeConfiguration\dependentAssembly\codeBase\{CCA70AFE-5E92-4B5E-940D-FAE11D564E72}]
+"name"="Microsoft.Build.Utilities.Core"
+"codeBase"="$BaseInstallDir$\MSBuild\14.1\Bin\Microsoft.Build.Utilities.Core.dll"
+"publicKeyToken"="b03f5f7f11d50a3a"
+"culture"="neutral"
+"version"="14.1.0.0"
```
Comment (rainersigwald) [inline src/Utilities/Microsoft.Build.Utilities.Core.pkgdef]: Does this version need to be updated? Ditto other `pkgdef`s.

**Comment by rainersigwald** [comment]
LGTM

File: CreateSetupPackageLayout.proj (hunk 4 above)
Comment (AndyGerlicher) [inline CreateSetupPackageLayout.proj]: Yes, there are also some shim files we have in OSS that are shipped but named differently. Will have to fix that up as well.

**Pull request merged**